### PR TITLE
Fixed deprecation warnings when using react

### PR DESCRIPTION
**Pull request by larsbs**

### CODE REVIEW

**Comment by BridgeAR** [comment]
@larsbs would you be so kind and fix the failing test? Is there no harmony flag or somthing similar for NodeJS 0.10 and 0.12?

Otherwise LGTM

File: package.json [hunk 1]
```diff
@@ -36,14 +39,15 @@
     "qejs": "^3.0.5",
     "ractive": "^0.6.1",
     "react": "^0.14.6",
+    "react-dom": "^0.14.7",
     "react-tools": "^0.13.0",
```
Comment (BridgeAR) [inline package.json]: Is `react-tools` still needed?

Comment (larsbs) [inline package.json]: No, it isn't. An about the failing tests, the problem is that react doesn't support anything below Node 4. I notice it after sending the pull request, so maybe this is a big drawback in order to accepting it.

Comment (BridgeAR) [inline package.json]: This is not true as far as I see it. The first time running this just takes _very_ long on 0.10 and 0.12 and therefor the first test fails. All tests pass if you increase the timeout to lets say 10 seconds for the first react test. It's about the NodeJS require call taking very long for the first file loaded with a react extension. I guess there are other things happening in the background in combination with react (I guess react is lazy loaded).

Comment (larsbs) [inline package.json]: Yes, you are right. This two lines confused me: https://github.com/facebook/react/blob/master/package.json#L60-L63

**Comment by rickyrobinson** [comment]
Hey guys, just wondering what the status of this PR is. Looks ok to me but some tests failing due to timeout? Any ideas what needs to be done before this fix can be merged?

**Comment by BridgeAR** [comment]
@larsbs please add the following to `./test/consolidate.js`:

```js
// Require the fixture here so the precompile happens before consolidate requires it
require('./fixtures/react/user.react');
```

That way the compilation is done before the tests run and they pass afterwards.

**Comment by larsbs** [comment]
Added

**Comment by niftylettuce** [comment]
see v.0.14.2

**Pull request closed without merging**